src/test/edu/colostate/cs/cs414/method_men/jungle/client/GameTest.java

```diff
@@ -16,9 +16,11 @@
     private NextMove n;
     private int row;
     private int col;
+    private boolean shh;
```

Comment:
Please change "shh" this to a more descriptive name like "debug" or if it is a not debug, call it something like "noDebugPrint"